tcp-transport/src/lib.rs

```diff
@@ -118,10 +123,6 @@ impl Transport for TcpConfig {
                             debug!("Incoming connection from {}", addr);
                             future::ok((TcpTransStream { inner: sock }, future::ok(addr)))
                         })
-                        .map_err(|err| {
-                            debug!("Error in TCP listener: {:?}", err);
-                            err
-                        })
                 })
```

Comment:
While on it, why not also add the `.listen(MAX_SIMULTANEOUS_CONNECTIONS)` [from the example](https://docs.rs/tk-listen/0.2.0/tk_listen/) - we could even make that configurable, putting the `TcpConfig` to good use finally.

Reply:
What `listen` does is handled by the `swarm`, although the swarm doesn't have a limit to the maximum number of connections at the moment and should have one.